dell-c6320/workflows/dell-secure-erase-drive-graph.json

```diff
@@ -0,0 +1,82 @@
+{
+  "friendlyName": "Secure Erase Drive",
+  "injectableName": "Graph.Drive.SecureErase",
+  "options": {
+    "finish-bootstrap-trigger": {
+      "triggerGroup": "secureErase"
+    },
+    "drive-secure-erase": {
+      "eraseSettings": null
+    },
+    "bootstrap-ubuntu": {
+      "triggerGroup": "secureErase",
+      "overlayfsFile": "secure.erase.overlay.cpio.gz"
+    }
+  },
+  "tasks": [
+    {
+      "ignoreFailure": true,
+      "taskName": "Task.Obm.Node.PxeBoot",
+      "label": "set-boot-pxe"
+    },
+    {
+      "waitOn": {
+        "set-boot-pxe": "finished"
+      },
+      "taskName": "Task.Obm.Node.Reboot",
+      "label": "reboot"
+    },
```

Comment:
Please keep aligned with other graph files with the order `label`->`taskName`->`waitOn`
```
{
   "label": "reboot",
   "taskName": "Task.Obm.Node.Reboot",
   "waitOn": {
      "set-boot-pxe": "finished"
    }
  },
```